services/todo-legacy/controllers/todo-controller.api.ts

```diff
@@ -0,0 +1,180 @@
+export const def = {
+  basePath: "/",
+  paths: {
+    "/todo": {
+      get: {
+        "x-operation-name": "getTodo",
+        parameters: [
+          {
+            name: "id",
+            in: "query",
+            description: "The todo ID.",
+            required: true,
+            type: "string",
+            format: "JSON"
+          },
+          {
+            name: "filter",
+            in: "query",
+            description:
+              "The criteria used to narrow down the number of customers returned.",
+            required: false,
+            type: "string",
```

Comment:
So far, we are using OpenAPI 2. IIRC, we eventually want to switch to OpenAPI 3. I guess the sooner the better?